src/python/pants/backend/python/rules/pex_from_target_closure.py

```diff
@@ -27,24 +28,36 @@ class CreatePexFromTargetClosure:
   additional_requirements: Tuple[str, ...] = ()
   include_source_files: bool = True
   additional_args: Tuple[str, ...] = ()
+  additional_input_files: Optional[Digest] = None
 
 
 @rule(name="Create PEX from targets")
-async def create_pex_from_target_closure(request: CreatePexFromTargetClosure,
-                                         python_setup: PythonSetup) -> Pex:
-  transitive_hydrated_targets = await Get[TransitiveHydratedTargets](BuildFileAddresses,
-                                                                     request.build_file_addresses)
+async def create_pex_from_target_closure(
+  request: CreatePexFromTargetClosure,
+  python_setup: PythonSetup,
+) -> Pex:
+  transitive_hydrated_targets = await Get[TransitiveHydratedTargets](
+    BuildFileAddresses,
+    request.build_file_addresses,
+  )
   all_targets = transitive_hydrated_targets.closure
   all_target_adaptors = [t.adaptor for t in all_targets]
 
   interpreter_constraints = PexInterpreterConstraints.create_from_adaptors(
     adaptors=tuple(all_target_adaptors),
     python_setup=python_setup
   )
-
+  input_files = []
+  merged_input_files: Optional[Digest] = None
```

Comment:
It would help to move this to line 57, right before the `if` block. That way this definition is much closer to where `merged_input_files` gets used on the next line.